sdk/core/azure-core/src/main/java/com/azure/core/http/ServiceVersion.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http;
```

Comment:
This doesn't need to be under .http, as we might also use this for amqp, etc. Same with the comment below.

Reply:
Consider perhaps putting this under .util